coinbaseadvanced/client_websocket.py

```diff
@@ -49,21 +65,21 @@ def _handle_message(self, ws, message: str):
         """
         data = json.loads(message)
 
-        if 'type' in data and data['type'] == 'error':
+        if data.get('type') == 'error':
```

Comment:
using `data.get(...)` instead of `'something' in collection and collection['something] == x` pattern.